tests/test_prob.py

```diff
@@ -150,11 +150,15 @@ def circuit(x, y, z):
     res_A = qml.jacobian(circuit_A)(*params)
 
     # Both jacobians should be of shape (2**prob.wires, num_params)
-    assert res_F.shape == (2 ** 2, 3)
-    assert res_F.shape == (2 ** 2, 3)
+    assert isinstance(res_F, tuple) and len(res_F) == 3
+    assert all(_r.shape == (2 ** 2,) for _r in res_F)
+    assert isinstance(res_A, tuple) and len(res_A) == 3
+    assert all(_r.shape == (2 ** 2,) for _r in res_A)
+    # assert res_F.shape == (2 ** 2, 3)
+    # assert res_A.shape == (2 ** 2, 3)
```

Comment:
how come this is commented out but not removed?

Reply:
Good catch!